recipes/libssh2/meta.yaml

```diff
@@ -29,8 +29,18 @@ test:
     - python {{ environ['PY_VER'] + '*' }}  # [win]
     
   commands:
-    - test -f $PREFIX/lib64/libssh2.a                 # [not win]
-    - if not exist %LIBRARY_LIB%\\libssh2.lib exit 1  # [win]
+    - test -f include/libssh2.h              # [not win]
+    - test -f include/libssh2_publickey.h    # [not win]
+    - test -f include/libssh2_sftp.h         # [not win]
+    
+    - test -f $PREFIX/lib64/libssh2.a        # [linux]
```

Comment:
I am not sure how to handle `lib64`. For now assuming only 64 bit builds on linux.